src/components/Proposal/Create/PluginForms/CreatePluginManagerProposal.tsx

```diff
@@ -627,7 +627,7 @@ class CreatePluginManagerProposal extends React.Component<IProps, IState> {
                       >
                         <option value="">Select a plugin...</option>
                         {plugins.map((plugin, _i) => {
-                          return <option key={`remove_plugin_${plugin.coreState.address}`} value={plugin.coreState.address}>{pluginNameAndAddress(plugin.coreState.address)}</option>;
+                          return <option id={`test-${_i}`} key={`remove_plugin_${plugin.coreState.address}`} value={plugin.coreState.address}>{pluginNameAndAddress(plugin.coreState.address)}</option>;
```

Comment:
can we avoid this `test` id  tag in the production code ?

Reply:
I saw this pattern across the project, maybe I'll open a PR to change all of them. For now I'll change this.